doc/src/Errors_details.rst

```diff
@@ -982,3 +982,52 @@ order of preference there are:
 - Send an email to ``developers@lammps.org``
 - Send an email to an :doc:`individual LAMMPS developer <Intro_authors>`
   that you know and trust
+
+.. _err0036:
+
+Neighbor list overflow, boost neigh_modify one
+----------------------------------------------
+
+The neighbor list code in LAMMPS uses a special memory allocation strategy
+to speed up building and accessing neighbor lists.
+
+Instead of making a memory allocation for each list of neighbors of the atoms
+LAMMPS allocates "pages" that have room for several neighbor lists.  This has
+two main advantages:
+
+#. It is not needed to first count how many neighbors there are for an
+   atom to determine the storage required.  Since the pages are much
+   larger than individual lists, LAMMPS just "fills up" the page until
+   there is not enough space left and then allocates a new page.
+
+#. There are fewer calls to the memory allocator functions (which can be
+   time consuming for long-running jobs and fragmented memory space) and
+   the resulting neighbor lists are close to each other physically which
+   improves cache efficiency.
+
+This is controlled by the two parameters "one" and "page", respectively,
+that can be set via the :doc:`neigh_modify command <neigh_modify>`. The
+parameter "one" is the estimate for the number of entries in a single
```

Comment:
This sounds a bit like the number of pairs in the entire neighbor list (extensive). Doesn't *one* specify the number of neighbors per atom (intensive)?

Reply:
Rephrased.